src/tokens/Assets.tsx

```diff
@@ -330,13 +330,16 @@ function AssetsScreen({ navigation, route }: Props) {
     return null
   }
 
-  const keyExtractor = (item: TokenBalance | Position | NftWithNetworkId[]) => {
```

Comment:
what about a regression test?

Reply:
Yes would be nice, was afraid it would be a time sink though given the complexity of RN’s list view rendering.